app/controllers/admin/loan_questions_controller.rb

```diff
@@ -14,12 +14,15 @@ def new
     field_set = CustomFieldSet.find_by(internal_name: 'loan_' + field_set_name)
     @loan_question = field_set.custom_fields.build
     authorize @loan_question
-    @loan_type_options = Loan.loan_type_options
```

Comment:
These changes were made by @linesarefuzzy. Do they still apply? See note in `_loan_types.slim`.